Proxy/server.js

```diff
@@ -1,13 +1,18 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const axios = require('axios');
 const parser = require('body-parser');
+const fetch = require('node-fetch');
+const fs = require('fs');
+const services = require('./services.js');
+const components = {};
 const app = express();
 const port = 7001;
 
 app.use(parser.json());
-app.use(morgan('dev'));
+// app.use(morgan('dev'));
```

Comment:
You should remove the comment and the require statement for morgan. It slows down the server.

Reply:
@kchen395, good catch. I'll make that change, push it and merge the pull request. Thanks!